strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/ConfigurationManagementControllerTest.java

```diff
@@ -465,16 +476,24 @@ public void testGetAndSetConfiguration()
         final String storageId = "storage2";
         final String repositoryId1 = "repository0";
 
+        Storage storage2 = new Storage(storageId);
+
         Storage storage1 = new Storage("storage1");
         client.addStorage(storage1);
         System.out.println("111111111111111");
```

Comment:
@yurygrishechkin : Could you please make sure you never commit this kind of dummy logging? Thanks!